ethcore/src/blockchain/blockchain.rs

```diff
@@ -177,6 +180,9 @@ impl BlockProvider for BlockChain {
 		self.query_extras_exist(hash, &self.block_details)
 	}
 
+	// We do not store tracing information.
+	fn have_tracing(&self) -> bool { false }
```

Comment:
just a stub for now - next PR for this feature will introduce processing logic into the Blockchain struct.